src/amo/components/Collection/index.js

```diff
@@ -124,6 +133,50 @@ export class CollectionBase extends React.Component<InternalProps> {
     );
   };
 
+  onSortSelect = (event: SyntheticEvent<any>) => {
+    const { clientApp, editing, filters, lang, router } = this.props;
+
+    event.preventDefault();
+
+    const sortValue = event.currentTarget.value;
+    const newFilters = {
+      ...filters,
+      sort: sortValue,
+    };
+
+    const pathname = oneLineTrim`/${lang}/${clientApp}
```

Comment:
It is a bit hard to parse to be honest.

Reply:
I converted it to a single line and prettier broke it up for me, so I guess that's how it will be.